rotkehlchen/chain/ethereum/defi/zerionsdk.py

```diff
@@ -128,6 +128,9 @@
     'Cozy',
     'Cozy • Yearn',
     'Cozy • Compound',
+    'Unagii',
```

Comment:
These 3 require frontend changes. Take a look for example here: https://github.com/rotki/rotki/pull/1991/files